sonar-project.properties

```diff
@@ -3,7 +3,7 @@ sonar.organization=interstellar
 sonar.python.coverage.reportPaths=.coverage-reports/coverage.xml
 sonar.coverage.exclusions=**/tests/**
 sonar.exclusions=.coverage-reports/**,docs/**
-sonar.python.version=3.8
+sonar.python.version=3.12
```

Comment:
As you can see [here](https://sonarcloud.io/project/issues?id=up42_up42-py&pullRequest=568&resolved=false&inNewCodePeriod=true) Sonar finds issues because it judges the code against the latest syntax, let's set this to 3.9